web-app/js/portal/details/SubsetItemsWrapperPanel.js

```diff
@@ -74,6 +89,7 @@ Portal.details.SubsetItemsWrapperPanel = Ext.extend(Ext.Panel, {
         this.spinnerToolItem = {
             id: 'spinnerToolItem',
             styles: 'fa-spin fa-spinner',
+            hidden: !layer.loading,
```

Comment:
Would it be safer to have `layer.loading !== false`? Just not sure what behaviour you want when `layer.loading` is either `undefined` or `null`.